tasks/clean_engine.yml

```diff
@@ -6,83 +6,78 @@
           password: "{{ vars['dr_sites_' + dr_source_map + '_password'] }}"
           ca_file: "{{ vars['dr_sites_' + dr_source_map + '_ca_file'] }}"
 
-    # Get all the running VMs and shut them down
-    - name: Fetch VMs in the setup
-      ovirt_vms_facts:
-          pattern:  status != down
-          auth: "{{ ovirt_auth }}"
-
-    - include_tasks: tasks/clean/shutdown_vms.yml vms={{ item }}
-      with_items: "{{ ovirt_vms }}"
-
-    - name: Update OVF_STORE disks for active storage domain
-      ovirt_storage_domains_facts:
-          pattern:  status = active and type != glance
-          auth: "{{ ovirt_auth }}"
+    - include_tasks: tasks/clean/shutdown_vms.yml storage={{ item }}
```

Comment:
I think this shouldn't be `shutdown_vms.yml`

Reply:
Why not? We want to shutdown the VMs which have disks reside on a storage domain.

Reply:
Ah sorry, you are right.